### ISSUE TITLE
Vulnerability in log4js dependency

### ISSUE THREAD

**Issue by ericcornelissen** [stryker-mutator/stryker-js#3373]
**Is your maintenance request related to a problem? Please describe.**
A CVE was published for log4js, leading `npm audit`, Snyk, and I'm sure other security tools to raise concerns about it through [the Stryker-js core package](https://github.com/stryker-mutator/stryker-js/tree/8b5455c30e09ece9636b8d36c46945c24d806cae/packages/core) (and possibly others, I didn't check).

Now, I don't _think_ this vulnerability is a real concern for Stryker users, nevertheless I think it's worth updating - even if just to err on the safe side.

**Describe the solution you'd like**
Update log4js to a recommended version, i.e. `6.4.0` or higher.

**Describe alternatives you've considered**
End-users can ignore the CVE, I don't think that's a good idea if it's not necessary. In this case, updating the vulnerable dependency _should_ be straightforward as no major version bump is required for the fix.

**Additional context**
References:
- [CVE-2022-21704](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21704)
- [SNYK-JS-LOG4JS-2348757](https://security.snyk.io/vuln/SNYK-JS-LOG4JS-2348757)
- [log4js changelog](https://github.com/log4js-node/log4js-node/blob/v6.4.0/CHANGELOG.md#640)

**Comment by nicojs** [comment]
Thanks for reaching out. We have been experiencing issues with upgrading log4js for some time, see https://github.com/log4js-node/log4js-node/issues/1037. But this issue bumps the priority to the top (priority of my spare time I can spend on my open source endeavours ;)).

**Comment by nicojs** [comment]
Fixed in  v5.6.1

```
$ npm i -D @stryker-mutator/core

added 185 packages, and audited 186 packages in 12s

30 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```
